os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_ota_ram.c

```diff
@@ -75,4 +75,121 @@ u32 OTA_Change(u32 OTAIdx)
 	return _TRUE;
 }
 
+#define TIZENRT_KERNEL_HEADER_LEN 0x10
+IMAGE2_RAM_TEXT_SECTION
+static u8 OTA_AddrChk(u8 idx, uint32_t pAddr, uint32_t *size, uint32_t *SBAddr)
+{
+	IMAGE_HEADER *Img2Hdr, *Img2DataHdr, *PsramHdr;
+	uint32_t ImgSize, SBheader;
+	u8 res = _TRUE;
+
+	/* Verify signature */
+	if(idx == 0) {
+		Img2Hdr = (IMAGE_HEADER *)(pAddr + TIZENRT_KERNEL_HEADER_LEN);
+	} else {
+		Img2Hdr = (IMAGE_HEADER *)pAddr;
+	}
+	if((Img2Hdr->signature[0] != 0x35393138) || (Img2Hdr->signature[1] != 0x31313738) || ((Img2Hdr->image_size) & 0x3)) {
+		res = _FALSE;
```

Comment:
There is no operations in `exit`.
So I think we can return directly.
```
return _FALSE
```
Then we can save the memory.

Reply:
Done modify.